spec/plaindatetime.html

```diff
@@ -28,19 +28,18 @@ <h1>Temporal.PlainDateTime ( _isoYear_, _isoMonth_, _isoDay_ [ , _hour_ [ , _min
       <p>
         This function performs the following steps when called:
       </p>
-      <emu-note>The value of ! ToIntegerWithTruncation(*undefined*) is 0.</emu-note>
       <emu-alg>
         1. If NewTarget is *undefined*, then
           1. Throw a *TypeError* exception.
-        1. Let _isoYear_ be ? ToIntegerWithTruncation(_isoYear_).
-        1. Let _isoMonth_ be ? ToIntegerWithTruncation(_isoMonth_).
-        1. Let _isoDay_ be ? ToIntegerWithTruncation(_isoDay_).
-        1. Let _hour_ be ? ToIntegerWithTruncation(_hour_).
-        1. Let _minute_ be ? ToIntegerWithTruncation(_minute_).
-        1. Let _second_ be ? ToIntegerWithTruncation(_second_).
-        1. Let _millisecond_ be ? ToIntegerWithTruncation(_millisecond_).
-        1. Let _microsecond_ be ? ToIntegerWithTruncation(_microsecond_).
-        1. Let _nanosecond_ be ? ToIntegerWithTruncation(_nanosecond_).
+        1. Set _isoYear_ to ? ToIntegerWithTruncation(_isoYear_).
+        1. Set _isoMonth_ to ? ToIntegerWithTruncation(_isoMonth_).
+        1. Set _isoDay_ to ? ToIntegerWithTruncation(_isoDay_).
```

Comment:
Nit: We should be more consistent about overwriting argument aliases (like this) vs. introducing new ones for mathematical values (as in Temporal.Duration two files up), and appropriate names for those new aliases in the latter case. I'll open an ECMA-262 issue.

Reply:
https://github.com/tc39/ecma262/issues/2967